packages/plugin-infra/src/utils/createInjectHooksRenderer.tsx

```diff
@@ -18,7 +18,11 @@ export function createInjectHooksRenderer<PluginDefinition extends Plugin.Shared
         const [ref, setRef] = useState<PluginWrapperMethods | null>(null)
         if (PluginWrapperComponent) {
             return (
-                <PluginWrapperComponent definition={plugin} ref={setRef}>
+                <PluginWrapperComponent
+                    definition={plugin}
+                    ref={(methods) => {
+                        if (methods) setRef(methods)
```

Comment:
Is this necessary?

Reply:
As tested, `methods` is swinging between `{ ... }` and `null`.